js/src/forum/components/LevelBar.js

```diff
@@ -7,17 +8,18 @@ export default class LevelBar extends Component {
 
     view() {
         const user = this.props.post.user();
+        const pointsText = app.forum.attribute('reflar-level-ranks.pointsText');
 
-        let expComments = (user.commentsCount() - user.discussionsCount()) * 21,
-            expDiscussions = user.discussionsCount() * 33;
+        let expComments = (user.commentCount() - user.discussionCount()) * 21,
+            expDiscussions = user.discussionCount() * 33;
 
         let expTotal = expComments + expDiscussions,
             expLevel = Math.floor(expTotal / 135),
             expPercent = (100 / 135) * ((expTotal) - (expLevel * 135));
 
         return (
             <div class="PostUser-level" title={expTotal + ' EXP'}>
-                <div class="PostUser-levelNumber">Level {expLevel}</div>
+                <div class="PostUser-levelNumber">{pointsText + ' ' + expLevel}</div>
```

Comment:
If anything, please do
```suggestion
                <div class="PostUser-levelNumber">{pointsText} {expLevel}</div>
```

Though that won't work with text that wants to be put on the right...

Reply:
In this case, the user could just use `margin-left: 20px` or `margin-right: 20px` to solve that

Reply:
...How? It's not a separate <span> tag or anything that the user could select and move around. They'd move the text with the level number.

Reply:
Right.... the text and the current level could be two separate elements wrapped in a div, but that could get more confusing....

Reply:
Use `<span>`

Reply:
@datitisev would separating the two elements then sticking them in a div work?

Reply:
Ok